src/main/kotlin/no/nav/faktureringskomponenten/service/PeriodiseringUtil.kt

```diff
@@ -0,0 +1,67 @@
+package no.nav.faktureringskomponenten.service
+
+import no.nav.faktureringskomponenten.domain.models.FakturaserieIntervall
+import no.nav.faktureringskomponenten.domain.models.FakturaseriePeriode
+import org.threeten.extra.LocalDateRange
+import java.time.LocalDate
+import java.time.YearMonth
+import java.time.temporal.ChronoUnit
+
+object PeriodiseringUtil {
+    private fun YearMonth.getQuarterStart(): YearMonth {
+        val monthInQuarter = (monthValue - 1) % 3
+        return minusMonths(monthInQuarter.toLong())
+    }
+
+    fun delIFakturerbarePerioder(
```

Comment:
Per nå har `FakturaGenerator` ansvar for å lage fakturerbare perioder og lage fakturaer ut av dem.
Hvis det er for mye ansvar for `FakturaGenerator` og fakturerbare perioder trenges et annet sted, må `FakturaGenerator` justeres til å reflektere dette.
Jeg tror på klasser med veldefinerte ansvarsområder, ikke så mye på ad hoc utilklasser.

Best å ikke introdusere utilklasser hvis mulig. Utilklasser er veldig ofte tegn på manglende modellering som samler logikk som bør være andre steder

Reply:
Enig. Men vi klarte ikke å justere på FakturaGenerator for å fikse problemet. 
Var meningen å skrive flere tester og gjøre mer refaktorering når v fikk ting til å fungere. 
Men nå har vi jo 2 andre tester som feiler å tenker da at ting må stilles på for å få alt til å funke.
Om vi klarer å få det til uten PeriodiseringUtil er det jo en bonus 🥇 Eller så må vi fjerne flytte på dette etterpå ja.